example/index.tsx

```diff
@@ -12,33 +12,35 @@ import {
   ListItem,
 } from '@chakra-ui/react';
 import { getTheme } from '../src/ui/theme';
+import usePublicationSW from '../src/ServiceWorker/client';
+import { pdfjs } from 'react-pdf';
 
-const pdfProxyUrl: string = process.env.CORS_PROXY_URL as string | undefined;
+const origin = window.location.origin;
 
-const AxisNowEpub = () => {
-  React.useEffect(() => {
-    // get content
-  }, []);
+// react-pdf web worker config with their default CDN version
+pdfjs.GlobalWorkerOptions.workerSrc = `${origin}/pdf-worker/pdf.worker.min.js`;
 
-  return (
-    <WebReader webpubManifestUrl="http://localhost:1234/samples/axisnow/encrypted/manifest.json" />
-  );
-};
-
-const SWTest = () => {
-  const [value, setValue] = React.useState<null | string>(null);
-  React.useEffect(() => {
-    fetch('http://localhost:1234/samples/pdf/degruyter.json').then((res) => {
-      res.text().then((text) => {
-        setValue(text);
-      });
-    });
-  });
-
-  return <div>{value}</div>;
-};
+const pdfProxyUrl = process.env.CORS_PROXY_URL as string | undefined;
 
 const App = () => {
+  usePublicationSW([
```

Comment:
For running the example app, I wonder if we should cache some things, but not everything, by default.  The purpose of the example app is to aid in development and I'd hate for something to actually be failing, but it looks like it's working because it's been cached.

Reply:
I disabled the SW in development, but yea not a bad idea. I can just cache one example for offline